RS-Project-6/SearchingAlgorithms/SearchAlgorithmsTest.java

```diff
@@ -0,0 +1,57 @@
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class SearchAlgorithmsTest {
+    LinearSearch linear = new LinearSearch();
+    BinarySearch binary = new BinarySearch();
+
+    @Test
+    public void testLinearSearch1() {

```

Comment:
Instead of writing the test method name with 1,2,3 - the name of test method should be relevant to what we are testing.